tests/spec/core/dfn-panel-spec.js

```diff
@@ -0,0 +1,168 @@
+"use strict";
+import { flushIframes, makeRSDoc, makeStandardOps } from "../SpecHelper.js";
+
+describe("Core — dfnPanel", () => {
+  afterAll(flushIframes);
+
+  const body = `
+    <section>
+      <h2>top level heading</h2>
+      <p><dfn>many</dfn>, <dfn>one</dfn>, <dfn>zero</dfn> references.</p>
+      <p>[=many=] [=many=] [=one=]</p>
+      <section>
+        <h3>nested section heading</h3>
+        <p>[=many=] [=many=]</p>
+      </section>
+      <p>[= many =]</p>
+    </section>
+  `;
+  const ops = makeStandardOps(null, body);
+
+  describe("dfnPanel state", () => {
+    it("opens panel on dfn click", async () => {
+      const doc = await makeRSDoc(ops);
+      expect(doc.getElementById("dfn-panel")).toBeNull();
+      const dfn = doc.querySelector("dfn");
+      dfn.click();
+      const panel = doc.getElementById("dfn-panel");
+      expect(panel).not.toBeNull();
+      expect(panel.classList.contains("minimized")).not.toBe(true);
```

Comment:
```suggestion
      expect(panel.classList).not.toContain("minimized")
```

Reply:
Got into habit of using `contains` as:
``` js
// el.classList == "foo-minimized-bar"
expect(el.classList).toContain("minimized") // pass
expect(el.classList.contains("minimized")).toBe(true); // fail
```

Should I change here?

Reply:
Couldn't confirm such behavior. Could you try it again? https://jsfiddle.net/saschanaz/3vt7ew15/1/

![image](https://user-images.githubusercontent.com/3396686/71075560-9cbc0e00-21c7-11ea-9c3a-26801e87a1b7.png)

Reply:
Whoa. Thanks for the test! Will update